scipy-1.0/paper.tex

```diff
@@ -225,10 +225,26 @@ \subsection*{Language choices}
 is now a requirement in our development workflow.
 
 \subsection*{API and ABI evolution}
-In general, we encourage changes that improve clarity in the API of the library,
-but strongly discourage breaking backwards compatibility, given our position near
-the base of the scientific Python computing stack.
-
+The application programming interface (API) for SciPy consists of
+approximately 2600 functions and classes. Adding new functionality has to
+meet the following criteria:
+\begin{itemize}
+    \item The algorithm is of relevance to multiple fields of science.
+    \item The implementation is based on a peer-reviewed article, and that article's importance is demonstrated (e.g. by having a significant number of citations).
```

Comment:
Minor: maybe generalize this a little bit:  it sounds as if we would not consider an algorithm which is classic enough be in textbooks.

Reply:
good point, done